.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+An example to help us reproduce the behaviour. (If you'd like some tips on the kind of information to include have a look at StackOverflow's [creating a Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve).)
```

Comment:
I'd rephrase that to say that MVCE is not an optional thing, but a required one if you report a bug. Currently it looks like a hint more.

Reply:
I was intending it more as a hint. 🤔 I'd prefer to have to ask for more information (and then close if no more information is offered) than to scare people off from submitting the initial report 😂